internal/core/autocomplete.go

```diff
@@ -196,7 +197,7 @@ func (node *AutoCompleteNode) isLeafCommand() bool {
 // BuildAutoCompleteTree builds the autocomplete tree from the commands, subcommands and arguments
 func BuildAutoCompleteTree(commands *Commands) *AutoCompleteNode {
 	root := NewAutoCompleteCommandNode()
-	scwCommand := root.GetChildOrCreate("scw")
+	scwCommand := root.GetChildOrCreate(os.Args[0])
```

Comment:
Yes But... we should not realy on `os.Args[0]` as this is a global varibale.
The binary name should be a parameter in BootstrapConfig and passed allong with context,